pygromos/files/gromos_system/ff/forcefield_system.py

```diff
@@ -0,0 +1,77 @@
+"""
+File:            forcefield managment for gromos_system
+Warnings: this class is WIP!
+
+Description:
+    This class handles all possible forcefields gromos_system accepts
+Author: Marc Lehner
+"""
+
+# imports
+import glob
+import importlib
+import collections
+
+from pygromos.files.topology.ifp import ifp
+from pygromos.files.topology.top import Top
+
+from pygromos.data import topology_templates
+from pygromos.data import ff
+from pygromos.data.ff import *
```

Comment:
UUUUUH! ^^ Could we formulate explicitly what is imported here?

Reply:
fixed